com.unity.netcode.gameobjects/Runtime/Messaging/RpcParams.cs

```diff
@@ -38,27 +17,22 @@ public struct ServerRpcParams
         public ServerRpcReceiveParams Receive;
     }
 
-    public struct ClientRpcSendParams : IHasUpdateStage
+    public struct ClientRpcSendParams
     {
-        private NetworkUpdateStage m_UpdateStage;
-
-        public NetworkUpdateStage UpdateStage
-        {
-            get => m_UpdateStage;
-            set => m_UpdateStage = value;
-        }
+        /// <summary>
+        /// ulong array version of target id list - use either this OR TargetClientIdsNativeArray
+        /// </summary>
         public ulong[] TargetClientIds;
```

Comment:
As you suggested in our discussion we can use `IReadOnlyList` here to also support `List<ulong>` here without any downsides.

Reply:
can it even be more restrictive and become `IEnumerable<T>` which only provides iteration but not index access?

Reply:
Wouldn't that be less restrictive?